docs/src/gs/create-workload-cluster.md

```diff
@@ -8,15 +8,19 @@ Choose one of the available templates for to create your workload clusters from
+| `OCI_WORKER_SHAPE_MEMORY_IN_GBS`          |                     | The amount of memory to be allocated to the <br/>instances. If <br/>not provided it is automatically computed by compute API for VMs, but you will need to set this if you want to use Bare Metal instances.                                                                                                                                                                                                                                                                          |
+| `OCI_WORKER_SHAPE_OCPUS`                  | 1                   | The number of OCPUs allocated to the instance                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `OCI_SSH_KEY`                             |                     | The public SSH key to be added to the Kubernetes nodes. It can be used to login to the node and troubleshoot failures.                                                                                                                                                                                                                                                                                                                                                                |
+| `OCI_CONTROL_PLANE_PV_TRANSIT_ENCRYPTION` | true                | [In-transit encryption](https://docs.oracle.<br/>com/en-us/iaas/Content/File/Tasks/intransitencryption.htm) provides a way to secure your data between instances and mounted file systems using TLS v.1.2 (Transport Layer Security) encryption. Only [some bare metal instances](https://docs.oracle.com/en-us/iaas/releasenotes/changes/60d602f5-abb3-4639-aa19-292a5744a808/) support In-transit encryption                                                                        |
+| `OCI_NODE_PV_TRANSIT_ENCRYPTION`          | true                | [In-transit encryption](https://docs.oracle.com/en-us/iaas/Content/File/Tasks/intransitencryption.htm) provides a way to secure your data between instances and mounted file systems using TLS v.1.2 (Transport Layer Security) encryption. Only [some bare metal instances](https://docs.oracle.com/en-us/iaas/releasenotes/changes/60d602f5-abb3-4639-aa19-292a5744a808/) support In-transit encryption                                                                             |
 
 The following Cluster API parameters are also available:
```

Comment:
Documentation error from previous commit: POD_CIDR cannot be 1.

Reply:
Can we please replace "TLS v.1.2" with just "TLS 1.2" so that screen readers don't make it sound awful? Also, the "i" in in-transit encryption should be lowercase.